.github/workflows/ci-core.yml

```diff
@@ -505,6 +505,7 @@ jobs:
       - name: Ensure clean after generate
         run: |
           git add --all
+          git diff --cached
```

Comment:
Did you mean to leave this? Let's update below if we need to, but it was left as `--stat` by design because it typically fails with many files and was spammy.

Reply:
This was actually necessary in order to debug why it failed, I can remove it tho